docs/user/files.rst

```diff
@@ -9,21 +9,36 @@ This works even if it has been changed in the meantime.
 
     The available options might be limited by :ref:`privileges`.
```

Comment:
```suggestion
    Option availability might be limited by :ref:`privileges`.
```
Is there a way to actually solve this?
"Availability of options" is also a bit iffy.
It is possible to cut "the" here, but it is also possible to misread that as the options that are available are limited (like now).